core/embed/rust/src/ui/model_mercury/component/share_words.rs

```diff
@@ -158,7 +157,7 @@ impl<'a> Component for ShareWords<'a> {
                     .text_font
                     .visible_text_height("1"),
             );
-        let ordinal = build_string!(3, inttostr!(ordinal_val), ".");
+        let ordinal = uformat!("{}.", ordinal_val);
```

Comment:
![image](https://github.com/trezor/trezor-firmware/assets/14001709/c60912e5-83d7-4a4b-a1b5-eb96bc077e0f)

Reply:
Well, somehow the `.` is not rendered.

Reply:
i think that was before the fixup? i forgot to add the `.` in one place

Reply:
Yes, it should be green if the CI runs again.